swagger_to_sdk_config.json

```diff
@@ -1,19 +1,66 @@
 {
-  "$schema": "https://raw.githubusercontent.com/lmazuel/swagger-to-sdk/master/swagger_to_sdk_config.schema.json",
-  "meta": {
-    "autorest_options": {
-      "version": "V2",
-      "csharp": "",
-      "reflect-api-versions": "",
-      "license-header": "MICROSOFT_MIT_NO_VERSION",
-      "sdkrel:csharp-sdks-folder": "./sdk",
-      "use": "@microsoft.azure/autorest.csharp@2.3.82"
-    },
     "advanced_options": {
       "create_sdk_pull_requests": true,
-      "sdk_generation_pull_request_base": "integration_branch",
-      "main_branch": "master"
+      "run_mode": "onePassMultipleReadmeMd" // or multiplePassSingleReadmeMd
+    },
+    "init_options": {
+      "init_script_path": "./eng/tools/sdk_init"
     },
-    "version": "0.2.0"
+    "generate_options": {
+      "generate_script_path": "./eng/tools/sdk_generate" // Param: <path_to_generate_input.json> <path_to_generate_output.json>
```

Comment:
Is the `<path_to_generate_output.json>` a required parameter or only if `parse_generate_output` is set to true?